core/templates/domain/classroom/classroom-backend-api.service.ts

```diff
@@ -51,14 +51,14 @@ export class ClassroomBackendApiService {
       });
 
     this.http.get(classroomDataUrl).toPromise().then((data: any) => {
-      this.topicSummaryObjects = data.topic_summary_dicts.map(
+      data.topic_summary_dicts = data.topic_summary_dicts.map(
```

Comment:
Do not redefine variables, especially fields on dicts. Use a separate local variable instead.

Reply:
Done.